keps/sig-node/2837-pod-level-resource-spec/README.md

```diff
@@ -1172,6 +1207,179 @@ back to aggregating container requests.
   size of the pod's cgroup. This means the pod cgroup's resource limits will be
   set to accommodate both pod-level requests and pod overhead.
 
+#### Hugepages
+
+With the proposed changes, hugepages-2Mi and hugepages-1Gi will be added to the pod-level resources section, alongside CPU and memory. The hugetlb cgroup for the
```

Comment:
only these two hugepages sizes specifically, or anything matching `hugepages-` prefix pattern that passes hugepages name and value validation (just like container limits/requests)?

https://github.com/kubernetes/kubernetes/blob/ee22760391bae28954a69dff499d1cead9a9fcf0/pkg/apis/core/validation/validation.go#L7099